_commands/basics/uptime.md

```diff
@@ -3,22 +3,45 @@
 
 uptime
 ------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`uptime` is used to check how long your Mac has been on since the last reboot
+
 ~~~ bash
-*short* example of command usage and output
+$ uptime
+11:31 up 1 day, 1:22, 2 users, load averages 1.97 2.04 2.14
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+Uptime gives a one line display for each of the following
+
+The current time:
```

Comment:
turn these into a bulleted list, also if you wanted the times shown to be on newlines that isn't showing up on the rendered file.